src/megatron/bridge/models/conversion/auto_bridge.py

```
@@ -51,6 +50,14 @@
 
 logger = logging.getLogger(__name__)
 
+try:
+    from modelopt.torch.quantization.utils import is_quantized
+except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)